containers/azure-terraform/test-project/create-backend.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+echo "create resource group"
```

Comment:
Great point, we can add an additional item in the backlog for this opportunity so that we can tie that additional contribution back and in the meantime we will get this PR out.